system/mp/mpcomm.cpp

```diff
@@ -756,8 +760,7 @@ protected: friend class CMPPacketReader;
                 // loop with short wait time and release CS to allow other side to proceed
                 StringBuffer epStr;
                 unsigned startMs = msTick();
-
-                unsigned loopCnt = ((CONNECT_READ_TIMEOUT / retrycount) / CONNECT_TIMEOUT_INTERVAL) + 1;
+                unsigned loopCnt = CONNECT_TIMEOUT_LOOPCNT;
```

Comment:
loopCnt is now 20x larger than before, right ?
I increased from 10 to 20 for more chances to check for other thread completing to speedup connection without wanting to affect overall timeout.